common/src/service/HttpClientHelper.cs

```diff
@@ -43,6 +43,10 @@ internal sealed class HttpClientHelper : IHttpClientHelper
 
         private readonly HttpClient _httpClientWithNoTimeout;
         private readonly HttpClientHandler _httpClientHandler;
+        private readonly DelegatingHandler _connectionLeaseTimeoutHandler;
+
+        // Consistent with track 2 guidance: https://github.com/Azure/azure-sdk-for-net/blob/2c338d26571f4ee84c3c280984af49bcb771fe2a/sdk/core/Azure.Core/src/Pipeline/ServicePointHelpers.cs#L27
+        private readonly TimeSpan connectionLeaseTimeout = TimeSpan.FromMinutes(5); 
```

Comment:
`_connectionLeaseTimeout`

Reply:
This field has been deleted, so I'll resolve this comment